climpred/bootstrap.py

```diff
@@ -278,24 +281,43 @@ def bootstrap_compute(
     ci_high_pers = 1 - p_pers / 2
 
     inits = hind.init.values
+    members = hind.member.values
     init = []
     uninit = []
     pers = []
+
+    if dim == 'member':
+        to_be_shuffled = members
+        shuffle_dim = 'member'
+    elif 'init' in dim:  # also allows ['init','member']
+        to_be_shuffled = inits
+        shuffle_dim = 'init'
+    else:
+        raise ValueError('Shuffle either `member` or `init`; not', dim)
     # resample with replacement
     # DoTo: parallelize loop
```

Comment:
Do you have an idea of how to do this? This seems "embarassingly parallel," but I don't know how to do this simply in python. Also this would be for another PR.

Reply:
I tried for many days months ago and didn’t succeed.